SPECS/shim-unsigned-aarch64/shim-unsigned-aarch64.spec

```diff
@@ -87,21 +23,31 @@ loading.
 
 %prep
 %autosetup -n shim-%{version} -p1
+# shim Makefile expects vendor SBATs to be in data/sbat.<vendor>.csv
+sed -e "s,@@VERSION@@,%{version}-%{release},g" %{SOURCE1} > ./data/sbat.mariner.csv
```

Comment:
Isn't Microsoft the vendor in this case? This way we could also re-use the `%{vendor}` macro.;)

Reply:
Fixed

Reply:
Actually, I will revert to using a hardcoded name. Linter is complaining that Vendor isn't `Vendor: Microsoft Corporation` so I cannot use %{vendor} in the file name...